crypto/dilithium/internal.h

```diff
@@ -0,0 +1,70 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0 OR ISC
+
+#ifndef AWSLC_HEADER_SIG_INTERNAL_H
```

Comment:
```suggestion
#ifndef AWSLC_HEADER_PQDSA_INTERNAL_H
```

Reply:
fixed in [7435f9b](https://github.com/aws/aws-lc/pull/1963/commits/7435f9b2920753efc71a6461c55be213dc24da65)